airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -49,14 +49,18 @@
         }
       },
       "customer_id": {
-        "title": "Customer Id",
+        "title": "Customer ID(s)",
         "type": "string",
-        "description": "Customer id must be specified as a 10-digit number without dashes. More instruction on how to find this value in our <a href=\"https://docs.airbyte.io/integrations/sources/google-adwords#setup-guide\">docs</a>"
+        "description": "Comma separated list of (client) customer IDs. Each customer ID must be specified as a 10-digit number without dashes. More instruction on how to find this value in our <a href=\"https://docs.airbyte.io/integrations/sources/google-adwords#setup-guide\">docs</a>",
+        "pattern": "^[0-9]{10}(,[0-9]{10})*$",
+        "examples": ["1234567890,0987654321"]
       },
       "login_customer_id": {
         "type": "string",
         "title": "Login Customer ID",
-        "description": "If your access to the customer account is through a manager account, this field is required and must be set to the customer ID of the manager account (10-digit number without dashes). More information about this field you can see <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">here</a>"
+        "description": "If your access to the customer account is through a manager account, this field is required and must be set to the customer ID of the manager account (10-digit number without dashes). More information about this field you can see <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">here</a>",
```

Comment:
Are you sure that for all of the listed `Customer ID(s)` one specified `Login Customer ID` will be suitable. Or will we need to make the `Customer Id => Login Customer ID" mapping?

Reply:
I had the same thought in the beginning but multiple (sub) customer accounts below 1 (manager) login_customer_id.
For another manager account a new connection should be created, otherwise it get's messy with the mapping.
https://developers.google.com/google-ads/api/docs/migration/login-customer-id